src/parsers/__tests__/in.test.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import parser from '../in'
+
+const trueCases = [
+  '"foo" in {"foo": "bar"}',
```

Comment:
You don't have to actually test for the values these will be evaluated to here. This file should merely test the parser.

Generators have separate test, which test for the code that is generated. Testing for different situations that don't have a syntactical difference should be handled there.

In this very case, actually the calculation of the result should be simply handled by the generated JavaScript program itself, so it's enough to test this with an integration test. For now, integration tests reside here:
https://github.com/kantord/emuto/blob/master/src/__tests__/interpreter.test.js#L326

This will also add some snapshot tests automatically, but the main point is testing the result.

Many time I implement the integration test first, and then implement the parser, then the generator, or anything else that is required. Usually I use the pattern mode in jest to only execute tests for the part that I'm working on, so that other failures don't annoy me. Once the integration test passes, that tells me that the feature is wholly implemented.

In this file it's enough to test an example of of the syntax (and dangerous edge cases) and an example of the resulting syntax tree. Here's an example: https://github.com/kantord/emuto/blob/master/src/parsers/__tests__/parentheses.test.js I believe the tests for this operator will not get any more complex than that of parentheses.